mappings/net/minecraft/world/EntityDetector.mapping

```diff
@@ -0,0 +1,29 @@
+CLASS net/minecraft/unmapped/C_kkiexczn net/minecraft/world/EntityDetector
+	FIELD f_etrsolop NON_CREATIVE_OR_SPECTATOR_PLAYERS Lnet/minecraft/unmapped/C_kkiexczn;
+		COMMENT Needs a better name
+		COMMENT TODO: Remove this comment!
+	FIELD f_hkridops SHEEP Lnet/minecraft/unmapped/C_kkiexczn;
```

Comment:
I think it's fine as-is, or `LIVING_SHEEP` to be slightly more precise

Reply:
the commented one is above :p

Reply:
Oh whoops